build.sbt

```diff
@@ -1,12 +1,36 @@
-name := "openie"
-
-organization := "edu.washington.cs.knowitall.openie"
-
-crossScalaVersions := Seq("2.11.8")
-
-scalaVersion <<= crossScalaVersions { (vs: Seq[String]) => vs.head }
+import org.allenai.plugins.CoreDependencies
+
+lazy val buildSettings = Seq(
+  organization := "org.allenai.openie",
+  crossScalaVersions := Seq(CoreDependencies.defaultScalaVersion),
+  scalaVersion <<= crossScalaVersions { (vs: Seq[String]) => vs.head },
+  publishMavenStyle := true,
+  publishArtifact in Test := false,
+  licenses += ("Open IE Software License Agreement", url("https://raw.githubusercontent.com/allenai/openie-standalone/master/LICENSE")),
+  homepage := Some(url("https://github.com/allenai/openie-standalone")),
+  scmInfo := Some(ScmInfo(
+    url("https://github.com/allenai/openie-standalone"),
+    "https://github.com/allenai/openie-standalone.git")),
+  pomExtra := (
+    <developers>
+      <developer>
+        <name>Michael Schmitz</name>
+      </developer>
+      <developer>
+        <name>Bhadra Mani</name>
+      </developer>
+      <developer>
+        <id>allenai-dev-role</id>
+        <name>Allen Institute for Artificial Intelligence</name>
+        <email>dev-role@allenai.org</email>
+      </developer>
+    </developers>),
+  bintrayPackage := s"${organization.value}:${name.value}_${scalaBinaryVersion.value}"
```

Comment:
This might not be needed anymore - I think I added it to the release plugin. You should see what `bintrayPackage` resolves to without this line.